extensions-core/druid-bloom-filter/src/test/java/org/apache/druid/query/filter/sql/BloomDimFilterSqlTest.java

```diff
@@ -273,10 +275,67 @@ public void testBloomFilters() throws Exception
     );
   }
 
+  @Test
```

Comment:
Per https://github.com/apache/druid/pull/6974#issuecomment-459578848, it'd be good to ignore this slow test

Reply:
Marked both as ignored